plasmapy/particles/ionization_state.py

```diff
@@ -389,9 +389,9 @@ def __eq__(self, other):
 
         Examples
         --------
-        >>> IonizationState('H', [1, 0], tol=1e-6) == IonizationState('H', [1, 1e-6], tol=1e-6)
+        >>> IonizationState('H', [1, 0], tol=1e-6) == IonizationState('H', [1, 1e-6], tol=1e-6)  # noqa: W505
```

Comment:
`W505` is the pycodestyle (via flake8) error code for maximum docstring line length.